guides.md

```diff
@@ -26,6 +27,20 @@ our values.
 - [Voicemail](#voicemail)
 - [References](#references)
 
+## GitHub
+
+## Managing Labels
+
+We have _synced repos_ configured that mirror all labels from the root `hyphacoop/organizing`, on each change to the root repo's labels. Though some of the below steps are more complicated than we'd ideally like, this is because the automation is cautious and won't delete in-use labels, so _nothing destructive will happen_.
```

Comment:
```suggestion
We have _synced repos_ configured that mirror all labels from the root `hyphacoop/organizing`,
on each change to the root repo's labels.
Though some of the below steps are more complicated than we'd ideally like,
this is because the automation is cautious and won't delete in-use labels,
so _nothing destructive will happen_.
```